src/Pages/MyCollection/MyCollection.tsx

```diff
@@ -0,0 +1,23 @@
+import styles from './MyCollection.module.css';
+import Phono_Logo from '../../assets/Phono_Logo';
+import SearchBar from '../../components/SearchBar/SearchBar';
+import CoverSwiper from '../../components/CoverSwiper/CoverSwiper';
+import AlbumInfo from '../../components/AlbumInfo/AlbumInfo';
+import NavBar from '../../components/NavBar/NavBar';
+
+export default function MyCollection() {
+  return (
+    <div className={styles.myCollectionPage}>
+      <Phono_Logo />
+      <SearchBar
+        placeholder={'Search my Collection'}
+        searchLocation={'myCollection'}
+      />
+      <CoverSwiper />
+      <AlbumInfo />
+      <div className={styles.navBar}>
```

Comment:
Can't you set this in the `NavBar` or pass it via a prop? `className` for example, which would be passed to the highest level element in that component